cmd/vic-machine/create/create.go

```diff
@@ -181,6 +182,12 @@ func (c *Create) Flags() []cli.Flag {
 		},
 
 		// volume
+		cli.StringFlag{
+			Name:        "default-volume-store, vsd",
+			Value:       "",
+			Usage:       "Specify the default volume store location, This location is used to support anonymous volumes such as image volumes. e.g.  \"datastore/path\" or \"datastore\".",
```

Comment:
`location, this location` or `location. This location`, or perhaps `location, which is`?